docs/discover/get-started-discover.asciidoc

```diff
@@ -0,0 +1,354 @@
+
+. To create a view of the document that you can bookmark and share, select ** View single document**.
+
+. To view documents that occurred before or after the event you are looking at, select
+**View surrounding documents**.
+
+
+
+
+[float]
+[[search-in-discover]]
+=== Search and filter data
+
+[float]
+==== Default mode: Search and filter using KQL
+
+One of the unique capabilities of **Discover** is the ability to combine
+free text search with filtering based on structured data.
+To search all fields, enter a simple string in the query bar.
+
+[role="screenshot"]
+image:images/discover-search-field.png[Search field in Discover]
+
+To search particular fields and
+build more complex queries, use the <<kuery-query,Kibana Query language>>.
+As you type, KQL prompts you with the fields you can search and the operators
+you can use to build a structured query.
+
+For example, search the ecommerce sample data for documents where the country matches US:
+
+. Enter `g`, and then select *geoip.country_iso_code*.
+. Select *:* for equals some value and *US*, and then click the refresh button or press the Enter key.
+. For a more complex search, try:
++
+```ts
+geoip.country_iso_code : US and products.taxless_price >= 75
+```
+
+[[filter-in-discover]]
+Whereas the query defines the set of documents you are interested in,
+filters enable you to zero in on subsets of those documents.
+You can filter results to include or exclude specific fields, filter for a value in a range,
+and more.
+
+For example, exclude results from the ecommerce sample data view where day of week is not Wednesday:
+
+. Click image:images/add-icon.png[Add icon] next to the query bar.
+. In the *Add filter* pop-up, set the field to *day_of_week*, the operator to *is not*,
+and the value to *Wednesday*.
++
+[role="screenshot"]
+image:images/discover-add-filter.png[Add filter dialog in Discover]
+
+. Click **Add filter**.
+. Continue your exploration by adding more filters.
+. To remove a filter, click the close icon (x) next to its name in the filter bar.
+
+[float]
+==== Search and filter using ES|QL
+
+You can use **Discover** with the Elasticsearch Query Language, ES|QL. When using ES|QL,
+you don't have to select a data view. It's your query that determines the data to explore and display in Discover.
+
+You can switch to the ES|QL mode of Discover from the application top bar.
+
+Note that in ES|QL mode, the **Documents** tab is named **Results**.
+
+Learn more about how to use ES|QL queries in <<try-esql, Using ES|QL>>.
+
+
+
+[float]
+[[save-discover-search]]
+==== Save your search for later use
+
+Save your search so you can use it later, generate a CSV report, or use it to create visualizations, dashboards, and Canvas workpads.
+Saving a search saves the query text, filters,
+and current view of *Discover*, including the columns selected in
+the document table, the sort order, and the {data-source}.
+
+. In the toolbar, click **Save**.
+
+. Give your search a title.
```

Comment:
I wanted to add , mabe we should mention that users can add a description ... but it seems we just use the description in the saved dialog? :)

Reply:
```suggestion
. Give your search a title and a description.
```